docs/HazardAnalysis/HazardAnalysis.tex

```diff
@@ -113,6 +113,54 @@ \section{Safety and Security Requirements}
 \wss{Newly discovered requirements.  These should also be added to the SRS.  (A
 rationale design process how and why to fake it.)}
 
+\begin{enumerate}[label=SR\arabic*.]
+
+     \item The system shall allow users to reset their password without logging in and without knowledge of their current password.
+    \item[] \emph{Rationale:} Users cannot be relied on to remember their login credentials correctly. There must be a method of allowing the user to access their account despite this.
```

Comment:
Also allows users to recover their accounts should a malicious third-party gain access to their accounts.